### PR TITLE
feat: adds VotiumBribeForwarder

### PR DESCRIPTION
**Pull request by doncesarts**
Adds VotiumBribeForwarder dial

### CODE REVIEW

File: tasks/deployEmissionsController.ts [hunk 1]
```diff
@@ -30,6 +31,10 @@ task("deploy-emissions-polly", "Deploys L2EmissionsController and L2 Bridge Reci
 
         const disperseForwarder = await deployDisperseForwarder(signer, hre)
         console.log(`Set PBAL bridgeRecipient to ${disperseForwarder.address}`)
+
+        const votiumBribeForwarder = await deployVotiumBribeForwarder(signer, hre)
+        console.log(`Set ? bridgeRecipient to ${votiumBribeForwarder.address}`)
```
Comment (doncesarts) [inline tasks/deployEmissionsController.ts:36]: which PL token is ?

File: tasks/utils/emissions-utils.ts [hunk 2]
```diff
@@ -188,12 +190,13 @@ export const deployL2BridgeRecipients = async (
 }
 
 export const deployDisperseForwarder = async (signer: Signer, hre: HardhatRuntimeEnvironment): Promise<DisperseForwarder> => {
+    const chain = getChain(hre)
+    const nexusAddress = resolveAddress("Nexus", chain)
+    const disperseAddress = resolveAddress("Disperse", chain)
     const mtaAddress = MTA.address
-    const constructorArguments = [mtaAddress]
+    const constructorArguments = [nexusAddress, disperseAddress, mtaAddress]
```
Comment (doncesarts) [inline tasks/utils/emissions-utils.ts:197]: I think `DisperseForwarder` is missing arguments on its constructor, please validate this.

File: tasks/utils/emissions-utils.ts [hunk 3]
```diff
@@ -204,6 +207,26 @@ export const deployDisperseForwarder = async (signer: Signer, hre: HardhatRuntim
     return disperseForwarder
 }
 
+export const deployVotiumBribeForwarder = async (signer: Signer, hre: HardhatRuntimeEnvironment): Promise<VotiumBribeForwarder> => {
+
+    const chain = getChain(hre)
+    const nexusAddress = resolveAddress("Nexus", chain)
+    const votiumBribeAddress = resolveAddress("VotiumBribe", chain)
+    const mtaAddress = MTA.address
+    const votiumBribeProposal = "TODO-SHOULD-BE-CONFIGURABLE"
```
Comment (doncesarts) [inline tasks/utils/emissions-utils.ts:216]: Where can this configuration can be set ?

**Review by alsco77 — APPROVED** [review]
LGTM

**Review by naddison36 — APPROVED** [review]

**Pull request merged**